config/software/cacerts.rb

```diff
@@ -20,10 +20,12 @@
 license_file "https://www.mozilla.org/media/MPL/2.0/index.815ca599c9df.txt"
 skip_transitive_dependency_licensing true
 
-default_version "2020-12-08"
+default_version "2021-01-19"
 
 source url: "https://curl.haxx.se/ca/cacert-#{version}.pem"
 
+# versions_list: https://curl.se/ca/cacert-#{version}.pem
```

Comment:
I think technically the version list is located in the HTML here: https://curl.se/docs/caextract.html

Reply:
Yes, that's right. Updating version list for URL  https://curl.se/docs/caextract.html